doc/man3/EVP_desx_cbc.pod

```diff
@@ -23,6 +23,10 @@ All modes below use a key length of 128 bits and acts on blocks of 128-bits.
 
 The DES-X algorithm in CBC mode.
 
+This algorithm is not provided by the default OpenSSL provider.
```

Comment:
Same here :)

Reply:
Sorry, I still don't get. It looks like it is just one algorithm here...

Reply:
Ignore my comment, I'm wrong.